migration/api.py

```diff
@@ -54,10 +62,11 @@ def get_next_page_params(resp: httpx.Response) -> dict[str, Any] | None:
         stop=stop_after_attempt(MAX_ATTEMPT_NUM),
         retry=retry_if_exception_type((httpx.HTTPError, JSONDecodeError)),
         reraise=True,
-        before_sleep=before_sleep_log(logger, logging.WARN)
+        before_sleep=before_sleep_log(logger, logging.WARN),
+        sleep=trio.sleep
```

Comment:
This is what `tenacity` says to do for compatibility with `trio`. If you don't include this, there are some warnings that pop up. However, there seems to be some typing issue right now with this, so you may see an error in your IDE.

https://tenacity.readthedocs.io/en/latest/#async-and-retry